src/solve.jl

```diff
@@ -20,23 +33,24 @@ function default_operator(system::InitialValueProblem{S}) where
 end
 
 """
-    default_operator(𝒮::InitialValueProblem{<:HybridSystem})
+    distribute_initial_set(system::InitialValueProblem{<:HybridSystem, <:LazySet)
 
-Return the default post operators for a given hybrid system.
+Distributed an initial state over all the modes of a hybrid system.
 
 ### Input
 
-- `𝒮` -- initial value problem of a hybrid system
+- `system` -- an initial value problem wrapping a mathematical system (hybrid)
+              and a set of initial states
 
 ### Output
 
-The pair `opC, opD` where `opC` is a continuous post-operator and `opD` is a
-discrete post-operator.
+A new initial value problem with the same hybrid system but where the set of initial
+states is the list of tuples `(state, X0)`, for each state in the hybrid system.
 """
-function default_operator(𝒮::InitialValueProblem{<:HybridSystem})
-    opC = BFFPSV18()
-    opD = LazyDiscretePost()
-    return opC, opD
+function distribute_initial_set(system::InitialValueProblem{<:HybridSystem, <:LazySet})
+    HS, X0 = system.s, system.x0
+    initial_states = [(state, X0) for state in states(HS)]
```

Comment:
Can we use `mode` or `loc(ation)` instead of `state`? (We need to use `states(HS)` of course.)